src/QMCHamiltonians/ReferencePoints.cpp

```diff
@@ -130,9 +130,9 @@ void ReferencePoints::write_description(std::ostream& os, std::string& indent)
   return;
 }
 
-void ReferencePoints::save(std::vector<ObservableHelper>& h5desc, hdf_archive& file) const
+void ReferencePoints::save(std::vector<ObservableHelper>& h5desc, hdf_path& enclosing_path, hdf_archive& file) const
 {
-  h5desc.emplace_back(hdf_path{"reference_points"});
+  h5desc.emplace_back(enclosing_path / "reference_points");
```

Comment:
Just feel the for-loop at 137 can be replaced with a more elegant range-based loop.

Reply:
I agree but since I don't want to write unit tests for dead code I'm going to leave it.

Reply:
fair enough